From 41d11cc1e8610c046e9889931197ced07aa195c1 Mon Sep 17 00:00:00 2001
From: Felix Yan <felixonmars@archlinux.org>
Date: Sun, 11 Jun 2017 13:40:20 +0800
Subject: [PATCH] Fix digitalocean plugin tests for Python 3

assertItemsEqual() doesn't exist in Python 3.x. Travis didn't fail because of some nose errors (so not all tests were run).
---
 .../certbot_dns_digitalocean/dns_digitalocean_test.py                | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/certbot-dns-digitalocean/certbot_dns_digitalocean/dns_digitalocean_test.py b/certbot-dns-digitalocean/certbot_dns_digitalocean/dns_digitalocean_test.py
index 7e97eed073..6ff76079b5 100644
--- a/certbot-dns-digitalocean/certbot_dns_digitalocean/dns_digitalocean_test.py
+++ b/certbot-dns-digitalocean/certbot_dns_digitalocean/dns_digitalocean_test.py
@@ -2,6 +2,7 @@
 
 import os
 import unittest
+import six
 
 import digitalocean
 import mock
@@ -133,8 +134,8 @@ def test_del_txt_record(self):
 
         correct_record_mock.destroy.assert_called()
 
-        self.assertItemsEqual(first_record_mock.destroy.call_args_list, [])
-        self.assertItemsEqual(last_record_mock.destroy.call_args_list, [])
+        six.assertCountEqual(self, first_record_mock.destroy.call_args_list, [])
+        six.assertCountEqual(self, last_record_mock.destroy.call_args_list, [])
 
     def test_del_txt_record_error_finding_domain(self):
         self.manager.get_all_domains.side_effect = API_ERROR
